### DISCUSSION TITLE
Detection Question: Detection is saying blantet is a person but baby is not

### DISCUSSION THREAD

**Discussion by MrCaspan** [blakeblackshear/frigate#15050] [Ask A Question] [answered]
Since I have turned on detection I have been trying to fine tune it but I am get stuff like the photo below. I have my baby's room camera that when she is actually in the bed it does not see a person, then she leaves the bed and her blanket gets detected as a person. I get that there can be false positives but the blanket gets 75% some times nad my actually baby gets like 40% or not seen as a person! Is there any way to fine tune this to help it?

I know the right thing to probaby do it just remove person detection form the room as it's an indoor camera and of course there will be a person in there and just put movement only for that camera. How do others deal with this if you want person detection in the place where false positives are happening all the time?

![image](https://github.com/user-attachments/assets/c5c94415-e0a9-407f-b32d-6aa93c4e767d)

**Comment by hawkeye217** [comment, marked as answer]
This is just because the default free model from Google that Frigate ships with is based on the COCO dataset, which does not primarily include security camera images (especially black & white / IR images).

Frigate+ aims to improve this.
